apps/communications/contacts/js/views/details.js

```diff
@@ -435,68 +425,18 @@ contacts.Details = (function() {
   };
 
   var renderSocial = function cd_renderSocial(contact) {
```

Comment:
Let's remove this by now, and wait until #open self-contained view will be ready in master to replace the whole code.